package.json

```diff
@@ -5,6 +5,7 @@
   "author": "Adobe",
   "license": "Apache-2.0",
   "scripts": {
+    "prepare": "husky install",
```

Comment:
This was recommended by Husky "to automatically have Git hooks enabled after install".